clang/test/dpct/cmake_migation/case_008/expected.txt

```diff
@@ -0,0 +1,9 @@
+find_package(IntelSYCL)
+find_package(IntelSYCL REQUIRED)
+find_package(IntelSYCL REQUIRED)
+#find_package(MPI REQUIRED)
+#find_package(OpenMP REQUIRED)
+dpct_find_package(cublas cufft nvToolsExt)
+find_package(DNNL REQUIRED dnnl)
+find_package(oneDPL REQUIRED)
```

Comment:
Have you verified these two mapping are the correct mappings for :
find_package(CUDNN REQUIRED cudnn)
find_package(CUB REQUIRED)